frontend/src/views/Character.vue

```diff
@@ -0,0 +1,233 @@
+    ]),
+    ...mapGetters(['getCharacterName','getCharacterPower', 'contracts', 'ownCharacters', 'getExchangeUrl']),
+    selectedCharacter(): Nft{
+      return this.characters[this.currentCharacterId];
+    },
+    haveCharacters(): boolean {
+      return this.ownedGarrisonCharacterIds.length > 0 || this.ownCharacters?.length > 0;
+    },
+    havePlazaCharacters(): boolean {
+      return this.ownCharacters?.length > 0;
+    },
+    canClaimGarrisonXp(): boolean {
+      return this.ownedGarrisonCharacterIds.filter((id: string|number) => +this.xpRewards[id] > 0).length > 0;
+    },
+  },
+  methods: {
+    ...mapMutations(['setCurrentCharacter']),
+    ...mapActions([
+      'mintCharacter',
+      'fetchMintCharacterPriceDecreasePerSecond',
+      'fetchCharacterMintIncreasePrice',
+      'fetchMintCharacterMinPrice',
+      'fetchMintCharacterFee'
+    ]) as StoreMappedActions,
+    ...mapGetters(['getExchangeTransakUrl']) as StoreMappedGetters,
+    toggleGarrison() {
+      this.garrison = !this.garrison;
+    },
+    async onMintCharacter() {
+      try {
+        await this.mintCharacter(this.mintSlippageApproved);
+      } catch (e) {
+        (this as any).$dialog.notify.error(i18n.t('plaza.couldNotMint'));
+      }
+    },
+    checkStorage() {
+      if (process.env.NODE_ENV === 'development') this.showAds = false;
+      else this.showAds = localStorage.getItem('show-ads') === 'true';
+    },
+    canRecruit() {
+      const cost = toBN(this.recruitCost);
+      const balance = toBN(+fromWeiEther(this.skillBalance) + +fromWeiEther(this.skillRewards));
+      return balance.isGreaterThanOrEqualTo(cost);
+    },
+    formatSkill() {
+      return fromWeiEther(this.skillBalance);
+    },
+    async updateMintCharacterFee() {
+      const recruitCost = await this.fetchMintCharacterFee();
+      const skillRecruitCost = await this.contracts.CryptoBlades.methods.usdToSkill(recruitCost).call();
+      this.recruitCost = new BN(skillRecruitCost).div(new BN(10).pow(18)).toFixed(4);
+    },
+  },
+  async mounted(){
+    this.checkStorage();
+  },
+  async created(){
+    this.mintPriceDecreasePerHour = new BN(await this.fetchMintCharacterPriceDecreasePerSecond()).div(new BN(10).pow(18)).multipliedBy(60*60).toFixed(6);
+    this.mintCharacterPriceIncrease = new BN(await this.fetchCharacterMintIncreasePrice()).div(new BN(10).pow(18)).toFixed(6);
+    this.mintCharacterMinPrice = new BN(await this.fetchMintCharacterMinPrice()).div(new BN(10).pow(18)).toFixed(4);
+    this.updateMintCharacterFee();
+    this.updateInterval = setInterval(async () => { await this.updateMintCharacterFee(); }, 2000);
+  },
+  components:{
+    BigButton,
+    CharacterList,
+    CharacterNav,
+    Character
+  }
+});
+</script>
+<style scoped lang="scss">
+
+.background-image {
+  background-image: url('../assets/artwork-fantasy-art-ruins.png') ;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top right;
```

Comment:
Can this have a min height for full view height. Can see the problem in the garrison tab.

Also can you apply a black opaque overlay over the image like in some of the other pages.

Reply:
fixed